mava/systems/jax/launcher.py

```diff
@@ -26,6 +27,10 @@ class NodeType:
     reverb = lp.ReverbNode
     courier = lp.CourierNode
 
+def copy_store(builder):
+    buildercopy = copy.copy(builder)
+    builder.store = copy.copy(builder.store)
```

Comment:
Did you meant here `builder.store = copy.copy(builder.store)`-> `builder.store = copy.copy(buildercopy.store)`

Reply:
Yes :) I copied from the wrong file. It should be fixed now :+1: